examples/uefi/libvirt.tf

```diff
@@ -0,0 +1,43 @@
+provider "libvirt" {
+    uri = "qemu:///system"
+}
+
+resource "libvirt_network" "dev-cluster" {
+   name = "dev-cluster"
+   domain = "dev-cluster.local"
+   mode = "nat"
+   addresses = ["10.0.200.0/24"]
+}
```

Comment:
Sure, not a problem, I will remove it. You are right, it it better to focus on what we really want to demonstrate here.